webpack.config.js

```diff
@@ -12,7 +12,7 @@ const licensesNotice = require('./jwplayer.license.notice.js');
 const compileConstants = {
     __SELF_HOSTED__: true,
     __REPO__: `''`,
-    __DEBUG__: true,
+    __DEBUG__: false,
```

Comment:
wait is this okay?

Reply:
Defaults should be for release and match commercial.